### PR TITLE
feat(template): improve report template style

### PR DESCRIPTION
**Pull request by vladanpaunovic**
Improving the visual design of the report template.

Note:
> I couldn't make a second graph (_Latency over time_) have a 100% width. This should be fixed in the future as currently, it won't show all 60 seconds of latency. I left a TODO comment in a code for this.

Screenshot 2021-10-05 at 14 26 03
Screenshot 2021-10-05 at 14 26 08
Screenshot 2021-10-05 at 14 26 12

### CODE REVIEW

File: template/css/dygraph.css [hunk 1]
```diff
@@ -47,6 +47,9 @@
 .dygraph-roller {
   position: absolute;
   z-index: 10;
+  border: 1px solid rgb(61, 61, 61);
+  border-radius: 3px;
+  text-align: center;
```
Comment (AbhiPrasad) [inline template/css/dygraph.css:52]: Why did we add this roller styling?

File: template/report.html.tmpl [hunk 2]
```diff
@@ -14,450 +15,596 @@
...
+                      <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">
+                        Total
+                      </th>
+                    </tr>
+                  </thead>
+                  {{ range .Latency }}
+                  <tr>
+                    <td class="px-6 py-4">{{ .Name }}</td>
+                    {{ if .Diff }}
+                      <td class="px-6 py-2">{{ round .Metrics.Min }} <div class="text-gray-400">({{ .Diff.Min }}%)</div></td>
```
Comment (AbhiPrasad) [inline template/report.html.tmpl:270]: Can we make the percentage darker than `gray-400`? It's kind of hard to see (assuming this follows the same deal as https://getbootstrap.com/docs/5.0/customize/color/). Maybe 600?

File: template/report.html.tmpl [hunk 3]
```diff
@@ -14,450 +15,596 @@
...
+          </div>
+        </details>
+      </section>
+
+      <div id="snackbar"></div>
+      <hr>
+        <footer class="p-4">
+          <p class="text-xs">Report generated with the <a class="text-blue-500" target="_blank"
+              href="https://github.com/getsentry/sentry-sdk-benchmark"><code>sentry-sdk-benchmark</code></a> tool.</p>
+        </footer>
```
[the remark this answers is not in the document]
Comment (vladanpaunovic) [reply template/report.html.tmpl]: awesome! Thanks!

File: template/report.html.tmpl (hunk 2 above)
Comment (vladanpaunovic) [reply template/report.html.tmpl:268]: Fine with me. Can we do these small improvements later? I am not sure I will have time

File: template/css/dygraph.css (hunk 1 above)
Comment (vladanpaunovic) [reply template/css/dygraph.css:52]: @rhcarvalho covered it [here](https://github.com/getsentry/sentry-sdk-benchmark/pull/43#discussion_r722212611). Basically Tailwind resets some styles

**Review by rhcarvalho — APPROVED** [review]
Getting this in as is and leaving small style tweaks for follow-ups so we can potentially re-generate reports with the new look.

Thanks @vladanpaunovic!

**Pull request merged**